llvm/test/CodeGen/NVPTX/load-store.ll

```diff
@@ -867,11 +1553,15 @@ define void @local_monotonic(ptr addrspace(5) %a, ptr addrspace(5) %b, ptr addrs
   ; CHECK: st.local.f64 [%rd{{[0-9]+}}], %fd{{[0-9]+}}
   store atomic double %f.add, ptr addrspace(5) %e monotonic, align 8
 
+  ; TODO: LLVM IR Verifier does not support atomics on vector types.
```

Comment:
Rephrase in terms what we need to fix or do. E.g. `add atomic vector loads/stores when LLVM verifier can deal with them`. Bonus point for including commented out sample of IR which triggers that verifier failure, so whoever has questions can easily reproduce it.

Reply:
Have redone the comments throughout to be clearer.